node/src/db/models/asset_states.rs

```diff
@@ -134,13 +162,13 @@ mod test {
         let digital_asset = DigitalAssetBuilder::default().build(&client).await?;
         let tari_asset_id = "asset-id-placeholder-0976544466643335678667765432355555555445544".to_string();
 
-        let mut additional_data_json = HashMap::new();
-        additional_data_json.insert("value", true);
+        let mut initial_data = HashMap::new();
+        initial_data.insert("value", true);
         let params = NewAssetState {
             name: "AssetName".to_string(),
             description: "Description".to_string(),
             asset_issuer_pub_key: PUBKEY.to_string(),
-            additional_data_json: serde_json::to_value(additional_data_json)?,
+            initial_data_json: serde_json::to_value(initial_data)?,
```

Comment:
could use `serde_json::json!()` macros:
```suggestion
        let params = NewAssetState {
            name: "AssetName".to_string(),
            description: "Description".to_string(),
            asset_issuer_pub_key: PUBKEY.to_string(),
            initial_data_json: serde_json::json!({"value": true}),
```

Reply:
All set, I've updated all the instances of these.